src/ResponseCache.php

```diff
@@ -61,6 +61,28 @@ public function shouldCache(Request $request, Response $response)
     }
 
     /**
+     * @param \Illuminate\Http\Request $request
+     *
+     * @return bool
+     */
+    public function shouldGetCachedResponse(Request $request)
+    {
+        if (!config('laravel-responsecache.enabled')) {
+            return false;
+        }
+
+        if ($request->attributes->has('laravel-cacheresponse.doNotCache')) {
+            return false;
+        }
+
+        if (!$this->cacheProfile->shouldCacheRequest($request)) {
+            return false;
+        }
+
+        return true;
```

Comment:
😅 
What about:
```php
return config('laravel-responsecache.enabled') 
    && ! $request->attributes->has('laravel-cacheresponse.doNotCache')
    && $this->cacheProfile->shouldCacheRequest($request);
```
**Notice** that it will not run the next check/validation if the previous fails.

--- 
also is this correct `'laravel-cacheresponse.doNotCache'` ?
- `cacheresponse` as no divisor between the 2 words neither camelCase
- `laravel-cacheresponse` lowercase with dashes to separate words
- `doNotCache` camelCase